e2e/tests/components/item-display-panel/item-display-panel.po.spec.ts

```diff
@@ -34,6 +34,10 @@ export class ItemDisplayPanelPage {
         return this.confirmClassExists(this.table1.$('tbody').$$('tr').get(index), 'highlight');
     }
     
+    waitForPanelToBeDisplayed() {

```

Comment:
Item display panel tests are still failing. Likely due to a modification required to these changes.

`browser.wait` is not a blocking function, so calling it doesn't actually perform any waiting. It instead returns a promise, so this function should be changed to:

```typescript
waitForPanelToBeDisplayed() {
   return browser.wait(protractor.ExpectedConditions.visibilityOf(this.panel1));
}
```

Anywhere that uses this function should then be changed to:

```typescript
it('should be possible to use the close button to close the panel', (callback) => {

...
    page.waitForPanelToBeDisplayed().then(() => {
          // add any checks in here that should be performed after the panel is open

         callback();
    });
...

});
```

This will also need changed for the typeahead list wait.